base/sysinfo.jl

```diff
@@ -139,6 +165,19 @@ mutable struct UV_cpu_info_t
     cpu_times!idle::UInt64
     cpu_times!irq::UInt64
 end
+
+"""
+    Sys.CPUinfo
+
+The `CPUinfo` type is a mutable struct with the following fields:
+- `model::String`: CPU model information.
+- `speed::Int32`: CPU speed. unit: MHz.
+- `cpu_times!user::UInt64`: Time spent in user mode. CPU state shows CPU time used by user space processes. unit: second.
```

Comment:
```suggestion
- `cpu_times!user::UInt64`: Time spent in user mode (in seconds). CPU state shows CPU time used by user space processes.
```

Reply:
It's not correct to say that these are in seconds, unfortunately.   Based on the `_show_cpuinfo` function, the units appear to be (a) in units of `seconds / Sys.SC_CLK_TCK` if `SC_CLK_TCK > 0`, and (b) in unknown units of `SC_CLK_TCK ≤ 0`

Reply:
since Its not correct to add unit to times, hence not adding the units for them